code/go/0chain.net/smartcontract/minersc/settings.go

```diff
@@ -207,6 +210,7 @@ func (gn *GlobalNode) set(key string, change string) error {
 			return fmt.Errorf("%s must be a hes string: %v", key, err)
 		}
 		gn.setKey(key, change)
+	case smartcontract.Cost:
```

Comment:
Not implemented yet?

Reply:
yeah, cost update is not implemented yet

Reply:
I was going to delegate it to someone